components/PageNavbar.tsx

```diff
@@ -5,7 +5,7 @@ export const PageNavbar = (props: { variant?: "light" }) => {
   const { variant } = props
 
   return (
-    <nav className={`flex justify-between ml-8 mr-8 mt-3 h-12 ${variant == "light" ? "text-black" : ""}`}>
+    <nav className={`flex justify-between ml-4 sm:ml-8 mr-4 sm:mr-8 mt-3 h-12 ${variant == "light" ? "text-black" : ""}`}>
```

Comment:
also very open to opinions about if the first screen breakpoint we want to target is actually `sm` or `md` ! 

tailwind references for this here: https://tailwindcss.com/docs/screens

Reply:
I think that `sm` would be mobile so what we're really aiming for here (at least as a priority) 👍 Thanks for the link to the docs thought 👍